README.md

```diff
@@ -153,50 +153,41 @@ look at a 'standalone' build, which hooks Eclipse OMR up to the its testing syst
 only.
 
 ### Basic configuration and compile
-To build standalone Eclipse OMR, run the following commands from the top of the
-source tree. The top of the Eclipse OMR source tree is the directory that contains
-run_configure.mk.
+To build standalone Eclipse OMR, run the following commands from the root of the
+source tree. For more detailed instructions please read `docs/BuildingWithCmake.md`.
```

Comment:
The path should be `doc/BuildingWithCMake.md` (`doc` not `docs`).

Also, can the file path here be made into a link?

Reply:
updated